tools/istio-iptables/pkg/cmd/root.go

```diff
@@ -39,6 +39,8 @@ var (
 	// Enable interception of DNS.
 	dnsCaptureByAgent = env.RegisterBoolVar("ISTIO_META_DNS_CAPTURE", false,
 		"If set to true, enable the capture of outgoing DNS packets on port 53, redirecting to istio-agent on :15053").Get()
+	dnsCaptureAll = env.RegisterBoolVar("ISTIO_META_DNS_CAPTURE_ALL",false,
```

Comment:
I anm not sure what's the diff with `ISTIO_META_DNS_CAPTURE`

Reply:
When use init-container to config iptables rules，
set ISTIO_META_DNS_CAPTURE true will only capturing DNS traffic to DNS server IP in /etc/resolv.conf.
set ISTIO_META_DNS_CAPTURE and ISTIO_META_DNS_CAPTURE_ALL treu will capturing all DNS traffic, even it not in /etc/resolv.conf.

Reply:
When use CNI config iptables rules, because CNI can not get DNS server IP in /etc/resolv.conf, set ISTIO_META_DNS_CAPTURE true will capturing all DNS traffic.

Reply:
That's really confusing literally

Reply:
How about `ISTIO_META_CAPTURE_ALL_DNS_TRAFFIC` ?